doc/source/whatsnew/v1.1.0.rst

```diff
@@ -623,57 +532,14 @@ The method :meth:`core.DataFrameGroupBy.size` would previously ignore ``as_index
 
    df.groupby("a", as_index=False).size()
 
-.. _whatsnew_110.api_breaking.apply_applymap_first_once:
```

Comment:
where did this one go?

Reply:
https://github.com/pandas-dev/pandas/pull/34865/files#diff-e763086372257ed7ca944d8bc5042a04R1002